### DISCUSSION TITLE
not able to add item in cart

### DISCUSSION THREAD

**Discussion by vjsmit** [learndevopsonline/learndevopsonline#1134] [Q&A]
Hi @r-devops ,

Not able to add to cart.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/63492214/2dc92a3f-e4a9-40c2-9882-f28f1ed494ea)

54.165.114.185 - ws

**Comment by maheshbusireddy** [comment]
Hi Vjsmit 
On which topic ypu are practicing.

**Reply by vjsmit** [reply]
terraform
